### PR TITLE
[extended-monitoring] Add description for the NodeDiskBytesUsage alert

### PR DESCRIPTION
**Pull request by meatuses**
## Description
Added reference for ncdu usage and a command to find pods with most disk usage to the alert description

## Why do we need it, and what problem does it solve?
Adds a place to start when investigating high disk usage. Adds a command to easily find pods with most disk usage on a node.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: extended-monitoring
type: chore
summary: Add description for the `NodeDiskBytesUsage` alert.
impact_level: low
```

### CODE REVIEW

**Comment by z9r5** [comment]
@kirkonru Please check this.

File: modules/340-extended-monitoring/monitoring/prometheus-rules/extended-monitoring/node.yaml [hunk 1]
```diff
@@ -21,6 +21,14 @@
       plk_markup_format: "markdown"
       plk_create_group_if_not_exists__node_disk_usage: "NodeDiskUsage,tier=cluster,prometheus=deckhouse,node={{ $labels.node }},kubernetes=~kubernetes"
       plk_grouped_by__node_disk_usage: "NodeDiskUsage,tier=cluster,prometheus=deckhouse,node={{ $labels.node }},kubernetes=~kubernetes"
+      description: |-
+        Node disk "{{$labels.device}}" on mountpoint "{{$labels.mountpoint}}" is using more than {{ printf "extended_monitoring_node_threshold{threshold=\"disk-bytes-warning\", node=\"%s\"}" $labels.node | query | first | value }}% of storage capacity.
+        Currently at: {{ .Value }}%
+
+        Retrieve disk usage info on the node: `ncdu -x {{$labels.mountpoint}}'
```
Comment (kirkonru) [inline modules/340-extended-monitoring/monitoring/prometheus-rules/extended-monitoring/node.yaml:28]:
```suggestion
        Retrieve the disk usage info on the node: `ncdu -x {{$labels.mountpoint}}'
```

File: modules/340-extended-monitoring/monitoring/prometheus-rules/extended-monitoring/node.yaml [hunk 2]
```diff
@@ -21,6 +21,14 @@
...
+
+        If output shows high disk usage in catalogue: /var/lib/containerd/io.containerd.snapshotter.v1.overlayfs/, following command will show pods with most usage:
+        `crictl stats -o json | jq '.stats[] | select((.writableLayer.usedBytes.value | tonumber) > 1073741824) | { meta: .attributes.labels, diskUsage: ((.writableLayer.usedBytes.value | tonumber) / 1073741824 * 100 | round / 100 | tostring + " GiB")}'`
       summary: |-
         Node disk "{{$labels.device}}" on mountpoint "{{$labels.mountpoint}}" is using more than {{ printf "extended_monitoring_node_threshold{threshold=\"disk-bytes-warning\", node=\"%s\"}" $labels.node | query | first | value }}% of storage capacity.
```
Comment (kirkonru) [inline modules/340-extended-monitoring/monitoring/prometheus-rules/extended-monitoring/node.yaml:33]:
```suggestion
        Node disk "{{$labels.device}}" on mountpoint "{{$labels.mountpoint}}" is using more than {{ printf "extended_monitoring_node_threshold{threshold=\"disk-bytes-warning\", node=\"%s\"}" $labels.node | query | first | value }}% of the storage capacity.
```

File: modules/340-extended-monitoring/monitoring/prometheus-rules/extended-monitoring/node.yaml [hunk 3]
```diff
@@ -21,6 +21,14 @@
       plk_markup_format: "markdown"
       plk_create_group_if_not_exists__node_disk_usage: "NodeDiskUsage,tier=cluster,prometheus=deckhouse,node={{ $labels.node }},kubernetes=~kubernetes"
       plk_grouped_by__node_disk_usage: "NodeDiskUsage,tier=cluster,prometheus=deckhouse,node={{ $labels.node }},kubernetes=~kubernetes"
+      description: |-
+        Node disk "{{$labels.device}}" on mountpoint "{{$labels.mountpoint}}" is using more than {{ printf "extended_monitoring_node_threshold{threshold=\"disk-bytes-warning\", node=\"%s\"}" $labels.node | query | first | value }}% of storage capacity.
+        Currently at: {{ .Value }}%
+
+        Retrieve disk usage info on the node: `ncdu -x {{$labels.mountpoint}}'
+
+        If output shows high disk usage in catalogue: /var/lib/containerd/io.containerd.snapshotter.v1.overlayfs/, following command will show pods with most usage:
```
Comment (kirkonru) [inline modules/340-extended-monitoring/monitoring/prometheus-rules/extended-monitoring/node.yaml:30]:
```suggestion
        If the output shows high disk usage in the /var/lib/containerd/io.containerd.snapshotter.v1.overlayfs/ directory, use the following command to show the pods with the highest usage:
```

File: modules/340-extended-monitoring/monitoring/prometheus-rules/extended-monitoring/node.yaml [hunk 4]
```diff
@@ -21,6 +21,14 @@
       plk_markup_format: "markdown"
       plk_create_group_if_not_exists__node_disk_usage: "NodeDiskUsage,tier=cluster,prometheus=deckhouse,node={{ $labels.node }},kubernetes=~kubernetes"
       plk_grouped_by__node_disk_usage: "NodeDiskUsage,tier=cluster,prometheus=deckhouse,node={{ $labels.node }},kubernetes=~kubernetes"
+      description: |-
+        Node disk "{{$labels.device}}" on mountpoint "{{$labels.mountpoint}}" is using more than {{ printf "extended_monitoring_node_threshold{threshold=\"disk-bytes-warning\", node=\"%s\"}" $labels.node | query | first | value }}% of storage capacity.
```
Comment (kirkonru) [inline modules/340-extended-monitoring/monitoring/prometheus-rules/extended-monitoring/node.yaml:25]:
```suggestion
        Node disk "{{$labels.device}}" on mountpoint "{{$labels.mountpoint}}" is using more than {{ printf "extended_monitoring_node_threshold{threshold=\"disk-bytes-warning\", node=\"%s\"}" $labels.node | query | first | value }}% of the storage capacity.
```

**Review by kirkonru — APPROVED** [review]
@z9r5 Some minor changes in the wording

**Comment by z9r5** [comment]
> @z9r5 Some minor changes in the wording

Thanks!

**Review by z9r5 — APPROVED** [review]

**Review by nabokihms — APPROVED** [review]

**Pull request merged**